### PR TITLE
Support python 3.11 on aarch64

### PR DESCRIPTION
**Pull request by mgu**
Fixes https://github.com/grpc/grpc/issues/30927

I've build the artifact on amd64 and tested the wheel on Mac M1.

I'm not sure about what do do with grpc_artifact_python_manylinux2014_aarch64.current_version

### CODE REVIEW

**Comment by mgu** [comment]
cc @gnossen

File: tools/dockerfile/grpc_artifact_python_manylinux2014_aarch64/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM dockcross/manylinux2014-aarch64:20210826-19322ba
+FROM dockcross/manylinux2014-aarch64:latest
```
Comment (gnossen) [inline tools/dockerfile/grpc_artifact_python_manylinux2014_aarch64/Dockerfile:15]: For reasons of reproducibilty, we prefer to pin to tags that are less likely to change.

**Comment by gnossen** [comment]
Super excited to see this PR! Thank you for the contribution.

**Comment by mgu** [comment]
Thanks for adding the tag, I was not sure because some images don't have one

**Comment by gnossen** [comment]
@mgu I think what you've already done is above and beyond the call of duty. The last bit will be fiddly CI-specific stuff that I'll take care of. Thank you very much for the contribution!

**Comment by ewdurbin** [comment]
Thanks so much to the many folks who were involved in seeing this through. This is a huge improvement in my day to day quality of life working on PyPI. 💜

**Pull request merged**